Dockerfile

```diff
@@ -18,17 +19,24 @@ RUN /env/bin/pip install \
   rm -rf /tmp/dea_ml
 
 # Below is the actual image that does the running
-FROM opendatacube/geobase:runner
+FROM opendatacube/geobase-runner:${V_BASE}
 ARG py_env_path=/env
 
+# Environment can be whatever is supported by setup.py
+# so, either deployment, test
+ARG ENVIRONMENT=deployment
+RUN echo "Environment is: $ENVIRONMENT"
```

Comment:
Why is this here?

Reply:
copy from the alchemist docker file to indicate the environment during building.